Public/Src/Engine/Processes/SandboxedProcessMac.cs

```diff
@@ -236,6 +242,14 @@ private async Task OnProcessStartedAsync()
                 LogProcessState($"IOException caught while feeding the standard input: {e.ToString()}");
                 await KillAsync();
             }
+            finally
+            {
+                // release the FileAccessManifest memory
+                // NOTE: just by not keeping any references to 'info' should make the FileAccessManifest object 
+                //       unreachable and thus available for garbage collection.  We call Release() here explicitly 
+                //       just to emphasise the importance of reclaiming this memory.
+                info.FileAccessManifest.Release();

```

Comment:
in `SandboxedProcess` (for Windows) there is already code that simply sets its `m_fileAccessManifest` field to null so that it can be garbage collected

Reply:
Does adding an explicit `info = null` hurt?

Reply:
You predicted my question

Reply:
I don't think it makes a difference; `info` is a local variable and assigning null to it just before that local variable is going to go out of scope seems pointless to me